taxonomy-editor-frontend/src/pages/allentries/index.jsx

```diff
@@ -0,0 +1,16 @@
+import useFetch from "../../components/useFetch";
+import EntriesList from "./EntriesList";
+
+const Entry = () => {
+    const {data: nodes, isPending, error} = useFetch('http://localhost:80/nodes');
+    // console.log(nodes, isPending, error);
+    return (
+        <div className="allentries">
+            {error && <div>{error}</div>}

```

Comment:
In general I'd prefer to split this... something like: 
`if (error) return ( <div className="allentries">{error} </div>`
and so on. That way even if the main content grows, those other simple cases (error, loading) are kept simple and visible